tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py

```diff
@@ -84,7 +84,7 @@ def testCanBeInitializedWithNonFloatNumberOfClusters(self):
     (1),
     (-32)
   )
-  def testCanBeInitializedWithNonFloatNumberOfClusters(self, number_of_clusters):
+  def testCannotBeInitializedWithNumberOfClustersLessThanOne(self, number_of_clusters):
```

Comment:
LessThanTwo?

Reply:
Indeed, `number_of_clusters=1` is not accepted either.